CMakeLists.txt

```diff
@@ -475,14 +475,38 @@ endif()
 # Cache this library's auto scanned sources for later reuse
 include(QuickCppLibCacheLibrarySources)
 
+# Dependencies needed to consume our cmake package
+set(PROJECT_PACKAGE_DEPENDENCIES [=[
+include(CMakeFindDependencyMacro)
+find_dependency(quickcpplib)
```

Comment:
Isn't `find_quickcpplib_library()` responsible for calling `find_package()`/`find_dependency()`? https://github.com/ned14/quickcpplib/blob/master/cmakelib/QuickCppLibUtils.cmake#L304

Reply:
This code gets injected into `llfioConfig.cmake` which cannot depend on `cmakelib` being loaded/used in the consumer project. If `find_quickcpplib_library()` behaviour is desired by the consumer it is on him to explicitly call `find_quickcpplib_library()` from his `CMakeLists.txt`. Is this problematic?

Reply:
Let's get things working for you first, then cpp-pm, then I'll figure out solutions to any other breakage which keeps the previous two happy.